### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by ganler**
- md5 checksum for models.
- Add steps to test download scripts in GitHub workflow.
- Enable fake building.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,8 +10,14 @@ jobs:
...
     - uses: actions/checkout@v1
-
-    - run: cmake . -DBUILD_TESTS=1 -DBUILD_LIB=0 -DBUILD_EXAMPLES=0 -DBUILD_USER_CODES=0 -DEXECUTABLE_OUTPUT_PATH=./bin
+    - run: sudo apt-get install libopencv-dev libgflags-dev # dependencies
+    - run: sh scripts/download-test-data.sh
+    - run: sh scripts/download-tinyvgg-model.sh
+    - run: sh scripts/download-openpose-thin-model.sh
+    - run: sh scripts/download-openpose-res50-model.sh
+    - run: sh scripts/download-openpose-coco-model.sh
+    - run: mkdir -p build && cd build
```
Comment (lgarithm) [inline .github/workflows/ci.yml:19]: this line is not used

File: cmake/helpers.cmake [hunk 2]
```diff
@@ -8,9 +8,16 @@ FIND_PACKAGE(OpenCV)
 FIND_PACKAGE(gflags)
```
Comment (lgarithm) [inline cmake/helpers.cmake:8]: OpenCV and gflags should also be `REQUIRED`

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -10,8 +10,14 @@ jobs:
```
Comment (lgarithm) [inline .github/workflows/ci.yml:10]: can we also add a CI for windows?

**Comment by lgarithm** [comment]
which editor are you using for coding?

**Comment by lgarithm** [comment]
looks good in general.

**Comment by ganler** [comment]
> which editor are you using for coding?

CLion.

File: .github/workflows/ci.yml (hunk 3 above)
Comment (ganler) [reply .github/workflows/ci.yml:10]: Yes, I am working on it. I just installed a windows system in my PC. I'll try this out these days.

**Pull request merged**